tests/e2e/test_packages_api.py

```diff
@@ -0,0 +1,44 @@
+import json
+from datetime import UTC, datetime
+from pathlib import Path
+from typing import Generator
+
+import pytest
+from fastapi.testclient import TestClient
+
+from dor.entrypoints.api.dependencies import get_inbox_path, get_pending_path
+from dor.entrypoints.api.main import app
+
+
+@pytest.fixture
+def package_test_client() -> Generator[TestClient, None, None]:
+    def get_inbox_path_override():
+        return Path("tests/output/test_packages_api/test_inbox")
+
+    def get_pending_path_override():
+        return Path("tests/fixtures/test_packages_api/test_pending")
```

Comment:
Nothing at this path yet, we'll need to figure out how we want to do handle this once the commented out pieces are completed, wired up.